frontend/src/components/common/Resource/DocsViewer.tsx

```diff
@@ -13,7 +13,6 @@ import Loader from '../Loader';
 
 const useStyles = makeStyles(() => ({
   root: {
-    height: 216,
```

Comment:
If we set a certain height to it then each tree view expands over the other present doc view, this is solved by giving a maxHeight to the DocViewer so that it scrolls as content grows.